src/java/org/apache/nutch/metadata/Nutch.java

```diff
@@ -80,4 +80,11 @@
 	public static final String STAT_PROGRESS = "progress";
 	/**Used by Nutch REST service */
 	public static final String CRAWL_ID_KEY = "storage.crawl.id";
+	
+	public static final String ARG_SEEDDIR = "url_dir";
+	public static final String ARG_CRAWLDB = "crawldb";
```

Comment:
Hi @lewismc I have added some documentation to the new introduced metadata. Please let me know if its proper. Thanks :)